src/libraries/Microsoft.PowerFx.Core/Public/Config/ReadOnlySymbolTable.cs

```diff
@@ -260,8 +260,9 @@ public IEnumerable<NamedFormulaType> SymbolNames
                 // GlobalSymbols are virtual, so we get derived behavior via that.
                 foreach (var kv in globals.GlobalSymbols)
                 {
-                    var type = FormulaType.Build(kv.Value.Type);
-                    yield return new NamedFormulaType(kv.Key, type);
+                    var type = FormulaType.Build(kv.Value.Type);
+                    var displayName = kv.Value.DisplayName != default ? kv.Value.DisplayName.Value : null;
```

Comment:
>kv.Value.DisplayName != default ? kv.Value.DisplayName.Value : null; [](http://example.com/codeflow?start=38&length=68)

Can we use ?. operator?

 kv.Value.DisplayName?.Value

Reply:
Since DName is struct, this syntax wouldn't work.